pdr_backend/dfbuyer/dfbuyer_agent.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, ppss: PPSS):
 
         # addresses
         batcher_addr = ppss.web3_pp.get_address("PredictoorHelper")
-        self.OCEAN_addr = ppss.web3_pp.get_address("Ocean")
```

Comment:
Change from `OCEAN_addr` -> `USDC_addr` not `USDC_address`. It's small, but we want to minimize change across the board. (And `addr` is shorter yet obvious, hence why it's used in many places here.)